src/components/HomePage/Banner.js

```diff
@@ -151,6 +153,23 @@ function renderInfo(classes) {
 	);
 }
 
+const Image = () => {
+	const data = useStaticQuery(graphql`
+    query {
+      placeholderImage: file(relativePath: { eq: "powellBackground.png" }) {
+        childImageSharp {
+          fluid(maxWidth: 300) {
```

Comment:
```suggestion
          fluid(quality: 100) {
```
When we specify the maxWidth, the PNG quality shrinks a lot and looks really fuzzy when you zoom in. Let's remove the maximum width parameter and also specify the image quality to be 100 so we get that the highest PNG quality possible.

Unfortunately this is just one of the downsides of PNG compared to SVG. Although the colors have retained the original design better, you'll notice that when you zoom in it's fuzzy. If you use an SVG however (for example the banner on the hack website), you'll notice that you do not get fuzzy edges when you zoom in.